docs/moment/04-displaying/07-difference.md

```diff
@@ -26,13 +26,13 @@ a.diff(b, 'days') // 1
 
 The supported measurements are years, months, weeks, days, hours, minutes, and seconds. For ease of development, the singular forms are supported as of **2.0.0**. Units of measurement other than milliseconds are available in version **1.1.1**.
 
-By default, `moment#diff` will return a number rounded towards zero (down for positive, up for negative). If you want a floating point number, pass `true` as the third argument. Before **2.0.0**, `moment#diff` returned a number rounded to nearest, not a rounded number rounded towards zero.
+By default, `moment#diff` will return a truncated number. If you want a floating point number, pass `true` as the third argument. Before **2.0.0**, `moment#diff` returned a number rounded to the nearest integer, not a number truncated towards zero.
```

Comment:
I think this is a good change, but I don't think "truncated toward zero" is a thing

Reply:
Yeah, I agree. I'll rebase it into one commit once you think it's clear.